tests/python/ngraph/test_corner_cases.py

```diff
@@ -51,6 +51,221 @@ def test_broadcast_mix_logic_op():
+        [0.25548967, 0.16348307, 0.73962755, 0.26804754, 0.53329105],
+        [0.85052903, 0.85473372, 0.24581613, 0.90716763, 0.80818399]],
+
+       [[0.54995721, 0.00509445, 0.24617661, 0.23970231, 0.3423044 ],
+        [0.92347134, 0.99974746, 0.07506943, 0.35241646, 0.471716  ],
+        [0.66744447, 0.40374308, 0.36575651, 0.6874286 , 0.52490164],
+        [0.31600314, 0.59628447, 0.91749988, 0.528848  , 0.25215523],
+        [0.87566073, 0.92525822, 0.59844993, 0.53071467, 0.46937456]],
+
+       [[0.40634484, 0.07199046, 0.41268365, 0.13759053, 0.26770341],
+        [0.56530379, 0.56539445, 0.20004284, 0.29760379, 0.15485124],
+        [0.55641568, 0.49869577, 0.96862646, 0.63867995, 0.13234088],
+        [0.88355243, 0.51987049, 0.10487715, 0.07591599, 0.81723034],
+        [0.17935192, 0.67845026, 0.5149324 , 0.77334196, 0.83712913]]], 
+        dtype=np.float32)
+
+    x = mx.symbol.Variable("x")
+    z = mx.sym.SoftmaxActivation(x)
+    forward, grad = run_softmax_forward_backward(z, input2)
+    true_forward = np.array(
+      [[[0.11626664, 0.24172528, 0.15222143, 0.26371348, 0.22607318],
+        [0.18126483, 0.24732885, 0.304981  , 0.12728041, 0.13914494],
+        [0.16663015, 0.16953874, 0.1586066 , 0.2369282 , 0.2682963 ],
+        [0.15499721, 0.22039092, 0.2170861 , 0.27755862, 0.12996715],
+        [0.15344891, 0.28748915, 0.1774283 , 0.18228401, 0.19934963]],
+
+       [[0.1281137 , 0.3200847 , 0.22353303, 0.14513712, 0.18313147],
+        [0.1311581 , 0.28239706, 0.14875129, 0.2770166 , 0.16067694],
+        [0.19869834, 0.1692418 , 0.14132744, 0.24230848, 0.24842395],
+        [0.265139  , 0.1997308 , 0.17299525, 0.16910373, 0.19303118],
+        [0.13156492, 0.16041236, 0.28331068, 0.24418832, 0.18052365]],
+
+       [[0.1383187 , 0.18156473, 0.18936853, 0.21395802, 0.27679   ],
+        [0.25277787, 0.1437927 , 0.13093017, 0.17491075, 0.29758847],
+        [0.1503702 , 0.1878821 , 0.1424408 , 0.24389109, 0.27541578],
+        [0.1704235 , 0.15544313, 0.27655905, 0.17257716, 0.2249972 ],
+        [0.2189498 , 0.21987234, 0.11959721, 0.23170872, 0.20987192]],
+
+       [[0.25881973, 0.15009509, 0.19101484, 0.18978216, 0.2102882 ],
+        [0.26962912, 0.29100007, 0.11542782, 0.15232135, 0.17162158],
+        [0.22752573, 0.17478591, 0.16827093, 0.23211835, 0.19729902],
+        [0.1581355 , 0.20929268, 0.2885733 , 0.1956441 , 0.14835447],
+        [0.23905596, 0.2512115 , 0.18117926, 0.16931345, 0.15923984]],
+
+       [[0.22951414, 0.16428624, 0.23097359, 0.17542477, 0.19980127],
+        [0.24256574, 0.24258773, 0.1683444 , 0.1855961 , 0.16090597],
+        [0.19252302, 0.18172522, 0.29073915, 0.20903045, 0.12598222],
+        [0.28281614, 0.19658898, 0.12981647, 0.12611076, 0.26466766],
+        [0.12840787, 0.21151797, 0.17961076, 0.23257242, 0.24789092]]],
+      dtype=np.float32)
+    assert np.allclose(forward, true_forward)
+    # Don't run backwards because mxnet is wrong for this shape and option
```

Comment:
NBD, but it would be nice if you could give a little more detail here (bug-tracker URL, or summary, etc.) in case people want to follow-up on this.  If I understand the comment correctly, this is a hole in our unit tests that ideally will be filled in at a later date.

Reply:
It's a hole in the unit tests of public MXNet for a deprecated op, they don't plan to support it moving forward, we're only doing this to make certain legacy models work correctly in inference. I'll expand the comment.